templates/rsyslog.conf.erb

```diff
@@ -48,6 +48,15 @@ $ActionSendStreamDriverAuthMode x509/name # authenticate by hostname
 $ActionSendStreamDriverPermittedPeer <%= @permitted_peer %>
 <% end -%>
 
+# File permissions
+
+<% if @umask %>
```

Comment:
need the `-` so that there aren't a bunch of blank lines added

so 

``` erb
<% if @umask -%>
```

and

``` erb
<% end -%>
```